Documentation/README_Interactable.md

```diff
@@ -0,0 +1,285 @@
+# Interactable
+![Interactable](/External/ReadMeImages/Interactable/InteractableExamples.png)
+
+## How to use Interactable
+Add the Interactable Component to a GameObject
+
+![Interactable](/External/ReadMeImages/Interactable/InteractableInspector_basicSteps.png)
+
+1. A Collider must exist on the GameObject with the Interactable or the child of the Interactable for it to receive input.
+2. Use the OnClick event to make something happen.
+3. Add visual feedback by linking a Target to a Profile and assigning a Theme.
+
+
+## Input Settings
+The basic features allow for button style interactions, such as pointer focus and clicks, that maps to interaction states to drive themes which are setup through the Interactable profile. Controller or hand focus, down, up and click (both near and far) are handled. Functionality can be extended using external scripts that can set state manually.
```

Comment:
Do we have any examples to point to for how to do this sentence? It seems really cool that it suggests a capability exists, and I think it would be helpful to have an example as well.

If this is going to be part of a next drop of docs that's okay (or if this actually refers to the stuff later in this file having that explicit would be nice)

Reply:
We do have an exmaple scene ‘InteractableExamples’

Reply:
Is it worth adding a diagram?